agent/logger/log_test.go

```diff
@@ -332,46 +332,109 @@ func TestSeelogConfig_FileLevelDefault(t *testing.T) {
 }
 
 func TestSetLevel(t *testing.T) {
```

Comment:
both log_test.go and log_windows_test.go do not have "unit" build tag (e.g. like this https://github.com/aws/amazon-ecs-agent/blob/master/agent/api/container/container_test.go#L1). this tag needs to be added, otherwise none of the tests in these files will be run in travis

Reply:
okay thanks, I added that in